### PR TITLE
Updated app with /api/topics endpoint

### PR DESCRIPTION
**Pull request by wakenado**

### CODE REVIEW

File: controllers/app.controllers.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const { fetchNewsTopics } = require("../models/app.models");
+
+exports.getNewsTopics = async (req, res) => {
+  const result = await fetchNewsTopics();
+  res.status(200).send(result);
```
Comment (Angaca) [inline controllers/app.controllers.js:5]: You should send the topics on a key of topics to live inside the body response --> see {topics}; we should send object in a not clear position inside the body

**Review by Angaca — CHANGES_REQUESTED** [review]
We need to send data in the body in a specific key, please review your controller. Test will need adjusting as well considering the new required position inside the key of topics in the body

File: error.js [hunk 2]
```diff
@@ -0,0 +1 @@
+exports.errorHandler = () => {};
```
[the remark this answers is not in the document]
Comment (wakenado) [reply error.js:1]: No problem. I'm on it.

File: controllers/app.controllers.js (hunk 1 above)
Comment (wakenado) [reply controllers/app.controllers.js]: Done it now.

**Pull request merged**